### DISCUSSION TITLE
✅ Enhanced Unit Control: Flexible Height and Block-Size Units in Elementor (dvh, svh, and lvh)

### DISCUSSION THREAD

**Discussion by maurocarreraok** [elementor/elementor#24405] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I've been using Elementor for a while now and have noticed that many users, including myself, frequently employ vh units. However, there are often issues with this, leading to unexpected behavior. It would be incredibly beneficial if Elementor could consider introducing a **new feature related to these units**, such as providing support or alternatives like dvh, svh, and lvh. Also, support for dvb, svb, and lvb for block-size units would be a valuable addition.

Compatible browsers: https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic

More information here:
https://www.youtube.com/watch?v=ru3U8MHbFFI

### Describe the solution you'd like

Add dvh, svh, and lvh to the units

**Comment by loucasl** [comment]
You can add any custom unit by clicking the pencil icon in the list of units

**Comment by nicholaszein** [comment]
Hey @maurocarreraok!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Core v3.10.0! 🥳 

As @loucasl correctly pointed out, you can use the Custom Units to add whatever units you want if they aren't available in the units list.

[Check the documentation](https://elementor.com/help/selecting-measurement-units/) for more info.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
